test/endtoend/unmanaged_tablet_test.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+source ./tools/test.env
+source ./test/endtoend/utils.sh
+
+# get_started_unmanaged:
+function get_started_unmanaged() {
+    echo "Apply latest operator-latest.yaml"
+    kubectl apply -f "operator-latest.yaml"
+    checkPodStatusWithTimeout "vitess-operator(.*)1/1(.*)Running(.*)"
+
+    echo "Apply 101_initial_cluster_unmanaged_tablet.yaml"
+    kubectl apply -f "101_initial_cluster_unmanaged_tablet.yaml"
+    checkPodStatusWithTimeout "example-zone1-vtctld(.*)1/1(.*)Running(.*)"
+    checkPodStatusWithTimeout "example-zone1-vtgate(.*)1/1(.*)Running(.*)"
+    checkPodStatusWithTimeout "example-etcd(.*)1/1(.*)Running(.*)" 3
+    checkPodStatusWithTimeout "example-vttablet-zone1(.*)1/1(.*)Running(.*)"
+
+    sleep 10
+    echo "Creating vschema and commerce SQL schema"
+
+    ./pf_vtadmin.sh > /dev/null 2>&1 &
+    sleep 5
+
+    waitForKeyspaceToBeServing commerce - 0
+    sleep 5
+
+    applySchemaWithRetry create_commerce_schema.sql commerce drop_all_commerce_tables.sql
+    vtctldclient ApplyVSchema --vschema-file="vschema_commerce_initial.json" commerce
+    if [[ $? -ne 0 ]]; then
+      echo "ApplySchema failed for initial commerce"
+      printMysqlErrorFiles
+      exit 1
+    fi
+    sleep 5
+
+    echo "show databases;" | mysql | grep "commerce" > /dev/null 2>&1
+    if [[ $? -ne 0 ]]; then
+      echo "Could not find commerce database"
+      printMysqlErrorFiles
+      exit 1
+    fi
+
+    echo "show tables;" | mysql commerce | grep -E 'corder|customer|product' | wc -l | grep 3 > /dev/null 2>&1
+    if [[ $? -ne 0 ]]; then
+      echo "Could not find commerce's tables"
+      printMysqlErrorFiles
+      exit 1
+    fi
+
+    insertWithRetry
+
+    assertSelect ../common/select_commerce_data.sql "commerce" << EOF
+Using commerce
+Customer
++-------------+--------------------+
+| customer_id | email              |
```

Comment:
Not necessarily in this Pull Request, but ideally we should extract some of the reused code into sharded shell functions. For instance all the code between line 28 and line 82 is mostly the same as in the four other tests. The function `get_started()` (in `utils.sh`) already extract all this code, but makes assumptions that are not always valid for every test, like the fact that vtorc is always here and that we expect a fixed number of vttablets.